src/tribol/common/Parameters.hpp

```
@@ -311,12 +311,14 @@ enum BasisEvalType
  */
 enum FaceGeomError
 {
-  NO_FACE_GEOM_ERROR,                          ///! No face geometry error
-  FACE_ORIENTATION,                            ///! Face vertices not ordered consistent with outward unit normal
-  INVALID_FACE_INPUT,                          ///! Invalid input
-  DEGENERATE_OVERLAP,                          ///! Issues with overlap calculation resulting in degenerate overlap
+  NO_FACE_GEOM_ERROR,                 ///! No face geometry error indicating an overlap intersection
+  NO_OVERLAP,                         ///! Indicates that no actual overlap exists, but does not indicate an error
```

Should we create a `FaceGeomInfo` struct for this?  Similar to `CaseInfo` and `EnforcementInfo`?

---

Maybe we just report `NO_FACE_GEOM_ERROR` when there is no overlap?

---

That is basically the idea...is that the only thing you care about is if there is an overlap or no overlap with no errors. I think a FaceGeomInfo would be more consistent, but would potentially introduce more logic/checks because I need to report an error and/or info. I'll leave as is for now because all the checks work nicely with checking for NO_FACE_GEOM_ERROR or != NO_FACE_GEOM_ERROR